package.json

```diff
@@ -35,12 +35,13 @@
     "compare-versions": "^3.5.1",
     "mdn-confluence": "^1.0.3",
     "ora": "^4.0.3",
+    "prettier": "^1.18.2",
     "yargs": "^15.0.1"
   },
   "scripts": {
     "confluence": "node ./node_modules/mdn-confluence/main/generate.es6.js --output-dir=. --bcd-module=./index.js",
     "lint": "node test/lint",
-    "fix": "node scripts/fix",
+    "fix": "node scripts/fix; prettier --write **/*.js **/*.ts **/*.md **/*.json",
```

Comment:
Can we limit this only to `.js` and `.ts` files for now? I have reservations about the way prettier rewrites Markdown and JSON and those will be harder to review. I have clearer expectations and higher confidence in the JS changes.

Reply:
All&nbsp;`*.json`&nbsp;files are&nbsp;ignored by&nbsp;default, except&nbsp;for&nbsp;`package.json` and&nbsp;files in&nbsp;the&nbsp;`schemas`&nbsp;directory. https://github.com/mdn/browser-compat-data/blob/5a688f80031e38714e5a6a865384e233707e869f/.prettierignore#L1-L3

Reply:
Done!

Reply:
@ExE-Boss It was the schemas in particular I was concerned about. There's some vertical white space for readability in the schemas and I'm not sure we should clobber that.

I suppose we could allow `package.json` though. I'd be OK with that.